### ISSUE TITLE
Update example files with DOAS to use current EnergyPlus best practice

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#7609]
Issue overview
--------------
There are several DOAS example files, but none of them use the best combination of terminal unit and other inputs, as far as I know.

Best practice is to use:
1. AirTerminal:SingleDuct:ConstantVolume:NoReheat (which is a little confusing) with the Design Specification Outdoor Air Object Name.
2. No return path to the DOAS airloop (no OA system, no OA controller, etc.) unless there's heat recovery, then I guess we need the return.
3. OA controller needs to line up with what the terminal unit is requesting.
4. Sizing:System method should be VentilationRequirement.

5ZoneAirCooledWithDOASAirLoop.idf - is close, but uses AirTerminal:SingleDuct:VAV:NoReheat with a fixed min flow fraction of 1.0
HVACTemplate DOAS systems are created with a similar configuration.
Many of the other DOAS examples are still using AirTerminal:SingleDuct:Uncontrolled which will get moved to AirTerminal:SingleDuct:ConstantVolume:NoReheat for v9.2, but the OA controls should be reviewed.

A documentation section should also be written - not sure where to put it.
And this exercise may reveal some needed enhancements to make this more straightforward.

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Prompted by Helpdesk ticket number 15312.

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by EnergyArchmage** [comment]
DOAS examples should also show how to set up central air handler controls for cold, neutral, and humidity-controlled DOAS supply air conditions.

**Comment by mjwitte** [comment]
@EnergyArchmage Good suggestions. Does "neutral" have a standard definition? That's always been a bit fuzzy to me.

**Comment by EnergyArchmage** [comment]
I don't know of a standard definition,  but here are some suggestions.

Neutral: supply air drybulb 18.3C setpoint, maximum humidity ratio 0.0093 setpoint, reheat after cooling coil does any latent control. 

Cold: supply air drybulb of 12.7C. 

Air-to-Air HX recovery on the OA system would also be good to demonstrate. 

Difficult to say but one could argue that for types of zone equipment that support AirTerminal:SingleDuct:Mixer (WAHP, PTHP, etc.) best practice is to use air terminal mixer feeding DOAS directly into zone equipment, rather than using AirTerminal:SingleDuct:ConstantVolume:NoReheat.

**Comment by wta12** [comment]
@EnergyArchmage: Could I know whether your suggestion coming from normal design rule for supply air condition ?
I am not sure if I can determine setpoint supply air conditions  as :
1.  Cold - For completely handling ventilation and zone latent load e.g ensure zone humidity setpoint  using DOAS design flow rate (might be hard in humid (A) climates). reheat should be limited as little as possible
2. Neural: DOAS Supply air condition is not add any load to zone e.g. have same dry-bulb temperature as zone setpoint (plus dewpoint  if humidity control requested)
Should load control type (sensible/ventilation/Total)  be an input for set point selection for each case ?

**Comment by EnergyArchmage** [comment]
yes,  I believe those suggestions are consistent with those design rules.  The Max humidity ratio and/or cold drybulb setpoint values may need to vary though when DOAS serves difficult space types, e.g. conference room.   E+ does not have latent load zone sizing so the supply air humidity condition needs to be determined at input, which is a bit difficult.    

DOAS AirLoopHVAC sizing is generally done using "VentilationRequirement."    A high latent load zone could also be designed with increased air flow from the DOAS over what the ventilation minimum flow rates from code suggests (rather than dry out the air for the entire system).  Since there is no latent zone sizing, there can be cases where one would need to manually increase the outdoor air specification to meet latent loads.

**Comment by wta12** [comment]
I think It's viable with EMS approach. Btw, It's absolutely a question about design purpose other than limitation of EP. I did not see a good "autosize" selection for coupling requirements.
